Documentation/DocuBlocks/Rest/Documents/post_create_document.md

```diff
@@ -239,5 +244,27 @@ Use of returnNew:
     logJsonResponse(response);
     db._drop(cn);
 @END_EXAMPLE_ARANGOSH_RUN
+
+@EXAMPLE_ARANGOSH_RUN{RestDocumentHandlerPostOverwrite}
+    var cn = "products";
+    db._drop(cn);
+    db._create(cn, { waitForSync: true });
+
+    var url = "/_api/document/" + cn;
+    var body = '{ "Hello": "World", "_key" : "lock" }';
+    var response = logCurlRequest('POST', url, body);
+    // insert
+    assert(response.code === 201);
+    logJsonResponse(response);
+
+    url = "/_api/document/" + cn + "?overwrite=true";
+    response = logCurlRequest('POST', url, body);
+    // insert same key
+    assert(response.code === 201);
```

Comment:
Should an overwrite actually return HTTP 201 (Created)? or should it rather return HTTP 200 (Ok)?
How can the caller distinguish between insert and replace here?

Reply:
The caller will get the old revision id in the returned object. Lets keep the options that modify the return code to a minimum so the API does not become unpredictable.